pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/model/XPathEvaluationException.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.util.fxdesigner.model;
+
+/** Exception during XPath evaluation. */
+public class XPathEvaluationException extends RuntimeException {
+
+    XPathEvaluationException(Throwable cause) {
```

Comment:
is package-private the intended visibility? or you just missed the access modifier?